devices/lidl/hg08673.json

```diff
@@ -175,6 +175,14 @@
         },
         {
           "name": "state/consumption"
+          "refresh.interval": 300,
+          "parse": {
+            "at": "0x0000",
+            "cl": "0x0702",
+            "ep": 0,
+            "eval": "item.val = 10 * Attr.val"
+          },
+          "default": 0
```

Comment:
The default can be safely removed, as it will be set already automatically. When removing, please do not forget the comma in the line above 😉